common/models/resource_consumer.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import bytesize "github.com/zpatrick/go-bytesize"
+
+type ResourceConsumer struct {
```

Comment:
As stated on another PR (h/t @zpatrick) , these models are not used as part of API requests/responses, so it seems like having them in `models` is not the right place.

Reply:
Oh, now I understand what you were getting at in that other comment @bivers. Is the idea, then, to just fold `ResourceConsumer` and `ResourceProvider` into `api/provider/aws/environment_scaler.go`?

Reply:
It could be, but to me it seems that those things are entirely independent of the `provider/aws` package. Doing a scaling algorithmn based entirely off of `ResourceProvider` and `ResourceConsumer` seems to be a discrete area of code that could be moved/tested independently.